src/script/chem/molfile/v2000.js

```diff
@@ -234,7 +234,7 @@ function parseCTabV2000(ctabLines, countsSplit) { // eslint-disable-line max-sta
 	const atomCount = utils.parseDecimalInt(countsSplit[0]);
 	const bondCount = utils.parseDecimalInt(countsSplit[1]);
 	const atomListCount = utils.parseDecimalInt(countsSplit[2]);
-	ctab.isChiral = !ctab.isChiral ? utils.parseDecimalInt(countsSplit[4]) !== 0 : true;
+	const isAbs = utils.parseDecimalInt(countsSplit[4]) === 1; // todo: check it
```

Comment:
What this TODO means? Please verify that it works properly and remove TODO

Reply:
Done